modules/schemas/pages/index.adoc

```diff
@@ -0,0 +1,38 @@
+= XML Schema files
+
+== Introduction
+
+The structure of eForms XML notices is defined in a set of XML Schema Definition (XSD) files. The schemas are based on Pre-Award document types of the https://docs.oasis-open.org/ubl/UBL-2.3.html[UBL 2.3 standard].
+
+== Structure of schema files
+
+The schema files are organised in 2 folders under `schemas`, following the same structure used by UBL:
+
+* `maindoc`: schemas for the individual document types.
+* `common`: schemas in the common library, referenced and used from the document types defined in the `maindoc` folder.
+
+The entry point to validate an XML notices against the schema is a one of the files in the `maindoc` folder. The appropriate file can be selected based on the root element, namespace, or notice subtype of the XML notice, using the information provided in the xref:notice-types:index.adoc[`notice-types.json` file].
+
+== Differences with UBL
+
+In order to meet some requirements specific to eForms, we have extended the UBL 2.3 standard, through the extension mechanism it provides, and also by creating an additional document type.
+
+This means that any valid eForms XML notice is also conforms to UBL 2.3.
+
+The files coming from the UBL standard (with names starting with `UBL` and `BDNDR`) have been left unmodified, with the exception of `UBL-ExtensionContentDataType-2.3.xsd`, which has been modified to reference only the eForms extension.
+
+The XSD files that are part of the UBL standard but are not used in eForms are not included in the eForms SDK.
+
+The XSD files added for eForms have a name that starts with `EFORMS`:
+
+`maindoc\EFORMS-BusinessRegistrationInformationNotice.xsd`::
+Specific document type for business registration notices (notice subtypes X01 and X02).
+
+`common\EFORMS-ExtensionApex-2.3.xsd`::
+Definition of the eForms extension element.
+
+`common\EFORMS-ExtensionAggregateComponents-2.3.xsd`::
+Definition of specific aggregate components for the eForms extension.
+
+`common\EFORMS-ExtensionBasicComponents-2.3.xsd`::
+Definition of specific basic components for the eForms extension.
```

Comment:
Maybe there could be a little section about the XML element order and linking to the fields.json "xsdSequenceOrder"?

Something like:

== XML element order

For convenience we provide the XML element order of fields and nodes in the `fields.json` via the "xsdSequenceOrder" array.

Reply:
This is covered in the "Working with the SDK" pages: https://docs.ted.europa.eu/eforms/latest/guide/xml-generation.html#_getting_the_xml_element_order_right

The purpose of this page is to describe the contents of the folder. The "how to use it" is I think a different topic.

I thought about adding a more general paragraph on the "design" and scope of the schemas: they define the structure and sequence of elements, but not a lot of the constraint to be followed. And mention there that it's usually better to use fields.json directly.
But this paragraph might overlap with the "Working with the SDK" pages, and can be added later anyways.